### DISCUSSION TITLE
Add CDA.PL to blocked service

### DISCUSSION THREAD

**Discussion by kiczorm** [AdguardTeam/AdGuardHome#3566] [Ideas]
Hi, 
I would like to ask to add this domain to blocked services. It's a streaming service like Netflix/Amazon Prime with a lot of legal movies but it's also used to share illegal (piracy) content and a lot of user materials which are not moderated and there is no age filtering inside.  

It's very popular in Poland.

It could be good to have the easy access to block it when you don't needed to prevent kids use it to often. 

Thanks,

**Comment by ainar-g** [comment]
Good day.  I'm not sure if our project has local services in its scope at the moment.  There is a feature request, #1692, which would allow people to add their own custom blocked services, but it's a matter of a long-term future.

In the meantime, you should be able to add its domains to your custom filtering rules section and, if you want to share it, put it somewhere (e.g. GitHub) so that people who would like to block it could do so simply by adding the URL to your filter list.
